### PR TITLE
Replaced page class to Json

### PR DESCRIPTION
**Pull request by diliger**

### CODE REVIEW

File: src/UserAdmin/Api/SettingsHandlers.cs [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
-        public static void Register() {
-            Handle.GET("/UserAdmin/admin/settings", (Request request) => {
+namespace UserAdmin
+{
+    public class SettingsHandlers
+    {
+        public static void Register()
+        {
+            Handle.GET("/UserAdmin/admin/settings", (Request request) =>
+            {
```
Comment (warpech) [inline src/UserAdmin/Api/SettingsHandlers.cs]: @diliger: It is a good practice to do code reformatting as a separate commit, that changes nothing else except from whitespace. This makes reviews and potential merge conflicts simpler.

File: src/UserAdmin/Program.cs [hunk 2]
```diff
@@ -8,12 +8,18 @@
...
+    {
         internal static string AdminGroupName = "Admin (System Users)";
         internal static string AdminGroupDescription = "System User Administrator Group";
 
-        static void Main() {
+        static void Main()
+        {
+            Application.Current.Use(new HtmlFromJsonProvider());
+            Application.Current.Use(new PartialToStandaloneHtmlProvider());
+
```
Comment (warpech) [inline src/UserAdmin/Program.cs]: Added some more explanation as the last section here: https://starcounter.io/replace-page-partial-classes-middleware/

Pls let me know if sth seems unclear

**Comment by warpech** [comment]
Ping. @diliger are you working on the requested changes?

**Comment by diliger** [comment]
Okay, I will take a look.

**Comment by diliger** [comment]
Done.

**Comment by warpech** [comment]
Assigning to myself to finish the review

**Comment by warpech** [comment]
Merged but to **develop**, not master as requested in the PR.
